docs/getting_started_guide/SIMP_on_Amazon_Web_Services.rst

```diff
@@ -0,0 +1,71 @@
+.. _gsg-simp_on_aws:
+
+SIMP on Amazon Web Services
+===========================
+
+This chapter provides notes and guidance on using the official SIMP Amazon
+Machine Image (AMI) to run the SIMP server in the cloud. 
+
+The SIMP AMI is built from the SIMP ISO, so much of the information contained 
+in the ISO installation section :ref:`ig-simp-server-installation` applies 
+here. 
+
+Provision a New EC2-Instance
+----------------------------
+
+To provision a new ec2-instance in the AWS cloud running the official SIMP AMI,
+follow these steps:
+   - Launch a new instance in the normal way, and navigate to the AWS 
+     Marketplace tab when prompted to choose a Machine Image. 
+   - Search the marketplace for the SIMP AMI, and locate the official published
+     SIMP AMI.
+   - Your ec2-instance should be **at least** ``t2.medium``, with 2 cpus and 
+     4GB of memory. Less than 4GB of memory will significantly slow down the 
+     bootstrapping process, and might cause problems in the future. See the 
+     AWS documentation_ for details on instance sizes.
```

Comment:
I was referring to the EC2 column on this page https://docs.puppet.com/pe/latest/sys_req_hw.html